static/css/components/work.less

```
@@ -255,14 +255,14 @@ div.editionTools {
     margin: 0 4px;
     border: 1px solid @lighter-grey;
     border-radius: 5px;
-    font-size: .8em;
+    font-size: @font-size-label-medium;
```

Ah, this should be body-size; the div is the label, it appears. That's some confusing html structure!

```suggestion
    font-size: @font-size-body-medium;
```